src/app/Http/Controllers/Api/LikeController.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace App\Http\Controllers\Api;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\LikeRequest;
+use App\Models\Like\Like;
+use Illuminate\Http\Request;
+
+class LikeController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        //
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(LikeRequest $request)
+    {
+        $data = $request->validated();
+
+        $oldLikes = Like::all()->where('movie_id', $data['movie_id'])->where('user_id', $request->user()->id);
+        if($oldLikes->count() > 0) {
+            $oldLike = $oldLikes->first();
+            if($oldLike->like == $data['like']) return response(1, 200);
```

Comment:
Pretpostavljam ovo je ako user lajkuje vec lajkovan movie.
Prvo sa FE bi trebalo onemoguciti da posalje poziv da lajkuje ako je vec lajkovan (disable button)
Zatim ako preko postmana proba treba da vrati 422 movie already liked. To mozes unutar LikeRequesta da sredis pogledaj
https://laravel.com/docs/9.x/validation#adding-after-hooks-to-form-requests